bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/DataLayerConfig.java

```diff
@@ -31,4 +31,12 @@
     @Property(label="Data Layer enabled")
     boolean enabled() default false;
 
+    /**
+     *
+     * @return {@code true} if the data layer client library is not included through the Core Page component,
+     * {@code false} otherwise. It defaults to {@code false} (client library included).
+     */
+    @Property(label="Data Layer client library not included")
+    boolean noClientlibIncluded() default false;
```

Comment:
I would name this `skipClientlibInclude` as I always get confused and have to think twice about the value of `<Something>` when we have something like `no<Something>=true`

Reply:
addressed in https://github.com/adobe/aem-core-wcm-components/pull/2252/commits/7cbbcddaca996a70288f0f6d4e8b8075de9b9b25